### DISCUSSION TITLE
p/hyper-v-vlan-trunking-for-pfsense/

### DISCUSSION THREAD

**Discussion by giscus** [donhector/donhector.github.io#23] [Giscus]
# p/hyper-v-vlan-trunking-for-pfsense/

Enabling trunking on a Hyper-V Network Adapter

https://hmolina.dev/p/hyper-v-vlan-trunking-for-pfsense/

**Comment by Neurology0443** [comment]
Just wanna say thank you for your blog post ! Alot of other post misguided the trunk part ! Thinking of it right now it seams logic to apply it on the pFsense VM LAN network adapter and not the guest VM itself!

I'm miles away from understanding networks in general, I'm more of a tinkerer and self-taught right now, soo your blog post helped me to get a bit closer to my goal :)

Thank you !

**Comment by donhector** [comment]
You are welcome! Glad it helped you!

**Comment by sgonzalezalbo** [comment]
GRACIAS por tu artículo, me ha ayudado mucho

**Comment by CJacobe92** [comment]
Thank you so much Hector for this article. It helped me a lot configuring PFSense on my Hyper-V.
It's frustrating to search for the right article on setting up VLAN's but your writing is the light among the shadows.
I hope you continue providing wonderful contents in the future!
